images.CI/shebang-linter.ps1

```diff
@@ -0,0 +1,34 @@
+$ErrorActionPreference = "Stop"
+
+function Validate-Scripts {
+    Param (
+        [Parameter(Mandatory=$true)]
+        [string[]]$Path,
+        [Parameter(Mandatory=$true)]
+        [string]$ExpectedShebang
+    )
+    $ScriptWithoutShebangLine = @()
+    Get-ChildItem $path -Recurse -File -Filter "*.sh" | ForEach-Object {
+        $shebangLine = Get-Content -Path $($_.FullName) | Select-Object -First 1
+        if ($shebangLine -eq $ExpectedShebang) {
+            Write-Host "Pattern '$ExpectedShebang' found in '$($_.FullName)'"
+        }
+        else {
+            Write-Host "Pattern '$ExpectedShebang' not found in '$($_.FullName)'"
+            $ScriptWithoutShebangLine += $($_.FullName)
```

Comment:
nitpick: Let's create relative path instead of FullPath?
e.g.:
`/home/runner/agent/images/linux/scripts/some.sh -> ./images/linux/scripts/some.sh`

Reply:
Thank you, I have implemented it.